### PR TITLE
Introduce HTMLObjectSection

### PR DESCRIPTION
**Pull request by gokaysatir**
Change-Id: I90719321781afcd29b72aa6d8eaafb186b4238dc

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by eszkadev — CHANGES_REQUESTED** [review]
I found 2 problems:
- it doesn't scale a button when zoomed in
- when button is shown and we zoom in/out - it shows button at wrong position

[dropdown-2024-04-23_22.40.37.webm](https://github.com/CollaboraOnline/online/assets/5307369/91ac7953-b3d4-4061-9029-0360fd3357c5)

File: browser/Makefile.am [hunk 1]
```diff
@@ -226,6 +226,7 @@ COOL_JS_LST =\
 	src/layer/tile/TilesSection.ts \
 	src/layer/tile/AutoFillMarkerSection.ts \
 	src/layer/tile/CellCursorSection.ts \
+	/src/layer/tile/HTMLObjectSection.ts \
```
Comment (hfiguiere) [inline browser/Makefile.am:229]: shouldn't this be `src/layer/tile/HTMLObjectSection.ts \` (no starting `/` ?

Comment (gokaysatir) [reply browser/Makefile.am]: Thank you for checking.

**Comment by eszkadev** [comment]
Thanks, now it works correctly with the zoom. But still it is very small (16x16), maybe we should scale that too?

**Review by eszkadev — APPROVED** [review]
code looks ok

**Comment by gokaysatir** [comment]
> Thanks, now it works correctly with the zoom. But still it is very small (16x16), maybe we should scale that too?

I agree, just couldn't be sure. Old code works that way - unscaled. Maybe we can consult @pedropintosilva and prepare a follow up? Thanks for the review btw :)

**Pull request merged**